blueprints/docker-for-mac.yml

```diff
@@ -79,6 +79,9 @@ services:
             "--swarm-default-advertise-addr=eth0",
             "--userland-proxy-path", "/usr/bin/vpnkit-expose-port",
             "--storage-driver", "overlay2" ]
+  # Monitor for image deletes and invoke a TRIM on the container filesystem
+  - name: trim-after-delete
+    image: "test"
```

Comment:
Good point -- what's the best way of handling the image? I think someone with access would need to create `linuxkit/trim-after-delete` and then I (or someone else) can run `make` which will `tag` and `push`?